ChainReactionTests/Items.feature

```diff
@@ -0,0 +1,207 @@
+﻿Feature: Items
+To have a quick access to products I would like to buy in the nearest future
+As a customer,
+I want to be able to add/delete product to/from my wishlist(s)
+A/C:
+1. I can add products to wishlist (I can see it in my profile)
+1.1 If there is an item in my wishlist, I can see it's name, price, selected option and amount of copies of this item.
+1.2 If there are several items in my wishlist, I can see all these items, their amount and cost (for each item and total).
+2. I can delete products from wishlist (I can't see in my profile)
+3. I don't see any products in an empty wishlist
+
+Background: 
+	Given I open browser
+	When I navigate to the page "http://www.chainreactioncycles.com/ru/en"
+	Then I see the main page is loaded
+
+	When I click on RubRub dropdown and I select language
+	And I click Update button
+	Then I see the main page is loaded
+	
+	When I click on Sign In on top menu
+	Then I see the Sign In page is loaded
+
+Scenario: Verify add good in wishlist
+	When I set following parameters on Sign In page
+		| Name     | Value                |
+		| email    | otbtguukb@emltmp.com |
+		| password | 123123123            |
+	And I click Sign In button
+	Then I see the main page is loaded
+
+	When I enter "Camelbak Podium Bottle 710ml" in the search text
+	And I click Search button
+	Then I see the search page is loaded
+
+	When I click on product with name "Camelbak Podium Bottle 710ml" from search results
+	Then I see the product page for "Camelbak Podium Bottle 710ml" is loaded
+
+	When I click Add To Wishlist button
+	Then I see add to wishlist popup is loaded
+	Then I wait for "1" seconds

```

Comment:
Why do we need to wait for 1 second? Can we remove this step?

Reply:
we can't, because poput isn't opening in time